package.json

```diff
@@ -98,6 +98,7 @@
     "opn-cli": "3.1.0",
     "prettier": "^1.19.1",
     "prop-types": "^15.6.0",
+    "react": "^16.14.0",
```

Comment:
This comes and goes. I'll create another PR later that should prevent it from showing up again.